packages/nodes-base/nodes/Google/Gmail/GmailTrigger.node.ts

```diff
@@ -32,16 +32,7 @@ export class GmailTrigger implements INodeType {
 		},
 		credentials: [
 			{
-				name: 'googleApi',
-				required: true,
-				displayOptions: {
-					show: {
-						authentication: ['serviceAccount'],
-					},
-				},
-			},
```

Comment:
why is this removed? can't service account and oauth live together even if we only use oauth?